### DISCUSSION TITLE
Welcome to bytedeck Discussions!

### DISCUSSION THREAD

**Discussion by tylerecouture** [bytedeck/bytedeck#873] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * We're new, so things are currently a bit lonely on this forum, please help us liven things up! 

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

Here's me!

I'm Tylere Couture (the extra 'e' was a birthday present I'm stuck with, not my fault! :)  I started Bytedeck as a summer project to learn Python and Django, and so I would have an LMS to use in my high school classroom back in [2015](https://github.com/bytedeck/bytedeck/commits/develop?after=df7ecf10cfa53c73577d8b1ce4d914efad9fea2e+2626).  The project has been continuously improved since then, mostly through feedback and suggestions from students.

I originally gave the project an open source license because 1. I didn't want my employer to be able to claim as theirs (my administrators and School District 72 have been very supportive and that concern was unfounded), and 2. my classroom almost exclusively uses free and open source (FOSS) software, including Linux!

In 2021 we created the Bytedeck Learning Society as a non-profit to steward the app. Many thanks to AviOG @mizzgoldberg for her part in that.

Okay, that was mostly about the app and not me, but Bytedeck has been one of my major passion projects for the past decade and I am thankful to @fjbagley and @aluxenburg for everything they have done and are doing to find funding, promote, improve, and generally give Bytedeck a healthy future!

Some stuff about me:
- I teach at [Timberline Secondary School](https://www.sd72.bc.ca/timberline/page/3408/digital-hackerspace) in the [Digital Hackerspace](https://hackerspace.bytedeck.com) (which includes courses in Digital Art and Design, 3D Modelling and Animation, Game Design, and Web Design, and Computer Programming) as well as Philosophy 12. Like Fergus, I use Bytedeck in my classroom everyday.  👩‍💻
- I coach e-sports 🕹️ and  [ball hockey](https://www.campbellrivermirror.com/sports/campbell-river-high-school-ball-hockey-team-shoots-its-way-to-provincial-semis/) at school, and coach ice hockey in the community. 🏒
- Before teaching I worked in physics, researching holographic diffusers.  👨‍🔬
- I've been active in the Canadian Armed Forces Primary Reserves for more than 20 years and spent some time in Kandahar.  🫡

Please don't hesitate to post questions; I love to talk about Bytedeck!  Ideas for improvement, best practices, use cases, technical support, whatever!  🙋

**Comment by aluxenburg** [comment]
Avi Luxenburg.
I am:
... a retired professor of education at Vancouver Island University;
... also a retired teacher from the Comox Valley School District;
... now on the board of Bytedeck Learning Society, where it is an honour to work with Tylere and Fergus;
... here to provide support to users regarding the use of Bytedeck with students/clients (whether through this forum, direct email, through learn.bytedeck, or in an online meeting if you wish.)

**Comment by fjbagley** [comment]
Hi fellow ByteDeck users,

I'm Fergus Bagley
I currently:
- teach in [Studio Tyee](https://youtu.be/EsDVYEaO2MA) (Digital Media) and PE at Carihi Secondary in Campbell River
- use ByteDeck every day with my students
- am on the board of ByteDeck Learning Society with Avi and Tylere

It has been great working with Avi v1, Avi 2.0 @aluxenburg and Tylere @tylerecouture. They all have unique skills and bring unique ideas to ByteDeck.   I'm looking forward to seeing what ideas you bring, how you use the LMS in your own way and watching ByteDeck grow as a result.

**Comment by karinatron** [comment]
Hello, I'm Karina Thygesen.

I'm currently a student in Vancouver Island University's Online Learning and Teaching graduate diploma, and a teacher in SD 71 (grades 4/5/6 Montessori).

I look forward to learning more about Bytedeck and how I can integrate it in my teaching practice.

p.s. Nice to see you, Avi!

**Reply by aluxenburg** [reply]
Wonderful to see your name come across my screen, Karina. 
I hope all is well with you.

**Comment by karinatron** [comment]
Nice to see your name too, Avi!
I hope you're enjoying retirement 😊


*I respectfully acknowledge that I live, work, and play on the unceded
traditional territory of the **K’ómoks and Pentlach First Nations,
including Sathloot, Sasitla, Ei’ksan, Puntledge, Cha’chae, and Tat’poos. *

**Comment by Malcommode** [comment]
Hello Bytedeck community!

I am Melanie Vogels and currently a student enrolled in the VIU Online Learning and Teaching Graduate Program. I am excited to learn about Bytedeck and would love to know if anyone is using it in a French Immersion or Core French setting?

Cheers,
Melanie

**Reply by tylerecouture** [reply]
Welcome @Malcommode!  You bring up something interesting, which is that the app is currently only English.  There are probably grants available for us to create a French language option for the app.

**Reply by aluxenburg** [reply]
So nice to see you here Melanie.
A couple of OLTD grads used an early version of a similar app that had quests and XP as their base model and engagement mechanism for French instruction. I happened to supervise their masters projects and papers on the very subject you bring up. 
They used something called Rezzly (which no longer exists). Having used both Rezzly and Bytedeck, I can tell you that, had Bytedeck been around for these OLTD students when they were completing the project, they would have used Bytedeck.
They used gamification to contextualize instruction, with a fun narrative to drive the learning.
Their papers should be on the VIU site. Corina Summerfelt and Carla Wilson.
Let me know if you have trouble finding the papers and I might be able to help.
Avi

**Comment by empyresse** [comment]
Hi! Bry Materi here from Vanderhoof. I'm a student in OLTD 504 as well - go Cohort 11!! 

Have a great day!
